### DISCUSSION TITLE
Dacs and slave machine

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#560] [General]
When using a slave machine to host vst plugins, is there a dac to ensure it has a quality dac or audio interface rather then the built in sound cards computers generally come with. Since it’s being streamed over the wire. Isn’t it all just 1s and 0s until it hits the audio gridder plugin window on the master machine controlling the plugin ??

**Comment by apohl79** [comment]
The audio data never hits your sound card or leaves the digital domain when processed via AG. AG is bridging the audio data from the DAW over a network connection to the the remote plugins and back into the DAW. AG server does not even require a sound card to be present on the computer.
